scripts/CSSRE/lib/py/kafka.py

```diff
@@ -87,14 +87,14 @@ def run_kafka_topics_script(self, namespace, kafka, topic="", filter="") :
       (apiobject): OpenShift API object with description of topics and partitions.
     """
     with self._oc.project(namespace), self._oc.timeout(self.settings.OC_TIMEOUT_5_MINUTES):
-
-      cmd = "-i svc/"+kafka+"-kafka-bootstrap -c kafka -- env - bin/kafka-topics.sh --bootstrap-server localhost:9096 --describe "
+      cmd = ["-i", "svc/" + kafka + "-kafka-bootstrap", "-c", "kafka", "--", "env", "-",
+             "bin/kafka-topics.sh", "--bootstrap-server", "localhost:9096", "--describe"]
       if topic:
-        cmd = cmd + " --topic " + topic
+        cmd += ["--topic", topic]
       if filter:
-        cmd = cmd + " --" + filter
+        cmd += ["--", filter]
```

Comment:
The `--` and `filter` will need to be appended as a single string. This way includes a space between them resulting in the script not picking up the script.

Reply:
Thanks for pointing that out, fixed now!